frontend/packages/data-portal/app/components/Run/ViewerPage.tsx

```diff
@@ -164,11 +164,83 @@ const isTopBarVisible = () => {
   return viewer?.uiConfiguration?.showLayerPanel.value ?? false
 }
 
-function ViewerPage({ run }: { run: any }) {
+const buildDepositsConfig = (annotations: any, tomograms: any[]) => {
+  const config: any = {}
+  const layers = currentNeuroglancerState().layers || []
+  for (const tomogram of tomograms) {
```

Comment:
Is this needed (associating tomograms to deposition) if we don't turn on/off the tomograms